cli-plugins/apps/command-reference/workload_get.hbs.md

```diff
@@ -4,7 +4,7 @@
 
 You can view workload details at whenever. Some details are:
 
- - Workload name and type.
+ - Workload name, type and namespace.
  - The source of the workload application.
  - The supply chain which took care of the workload.
  - The supply chain resources which interact with the workload. It also has the output of the resource stamped out by the supply chain
```

Comment:
The specific resources within the supply chain which interacted with the workload and the stamped out resources associated with each of those interactions.